source/features/mark-merge-commits-in-list.tsx

```diff
@@ -0,0 +1,63 @@
+import './mark-merge-commits-in-list.css';
+import select from 'select-dom';
+import gitmerge from 'octicon/git-merge.svg';
+import * as api from '../libs/api';
+import features from '../libs/features';
+import {getRepoGQL} from '../libs/utils';
+
+const getMergeCommits = async (commits: string[]): Promise<string[]> => {
+	const {repository} = await api.v4(`
+		repository(${getRepoGQL()}) {
+			${commits.map((commit: string) => `
+				${api.escapeKey(commit)}: object(expression: "${commit}") {
+					... on Commit {
+							parents{
+								totalCount
+							}
+						}
+					}
+			`).join('\n')}
+		}
+	`);
+
+	return filterCommits(repository);
+};
+
+function filterCommits(commits: object): string[] {
+	const mergeCommits = [];
+	for (const commit of Object.entries(commits)) {
+		if (commit[1].parents.totalCount === 2) {
+			mergeCommits.push(commit[0]);
```

Comment:
The current behavior is “given a list of commit hashes, return a list of commit hashes *as escaped by the API.*” The second part feels out of place.

Functions should try to be self-contained. Here it’s returning the escaped API key instead of a regular commit. Use this to unescape it

```suggestion
			mergeCommits.push(commit.slice(1));
```

As mentioned earlier, `filterMergeCommits` receives a list of commit hashes and returns a list of merge commit hashes. Such function would make the code more understandable.